src/workloads/scale/LoadTest.yml

```diff
@@ -0,0 +1,119 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  Based on LongLivedTransactions Insert workload. Using to experiment with bulk load for PERF-2330
+
+GlobalDefaults:
+  dbname: &dbname perf2330
+  MaxPhases: &MaxPhases 20
+
+  # The Sample Document Shape.
+  Document: &Doc
+    ts: {^Now: {}}
+    caid: {^RandomInt: {min: 0, max: 1000}}
+    cuid: {^RandomInt: {min: 0, max: 100000}}
+    prod: {^RandomInt: {min: 0, max: 10000}}
+    price: {^RandomDouble: {min: 0.0, max: 1000.0}}
+    data: {^Join: {array: ["aaaaaaaaaa", {^FastRandomString: {length: {^RandomInt: {min: 0, max: 10}}}}]}}
+    c: &string {^RandomString: {length: 75}}  # Adjust this so the doc comes out as 200 B.
+
+  FirstIndex: &FirstIndex
+    - keys: {price: 1, ts: 1, cuid: 1}     # Ptc
+
+  MoreIndexes: &MoreIndexes
+    - keys: {price: 1, cuid: 1}            # Pc
+    - keys: {caid: 1, price: 1, cuid: 1}   # Cpc
+
+  # Loader Config.
+  LoadThreads: &LoadThreads 8
+  CollectionCount: &CollectionCount 8
+  LoadBatchSize: &LoadBatchSize 1000
+
+  InitialDocumentCount: &InitialNumDocs 5000000
+  SecondDocumentCount: &SecondNumDocs 3000000
+  FinalDocumentCount: &FinalDocumentCount 1000000
+
+Clients:
+  Default:
+    QueryOptions:
+      socketTimeoutMS: -1
+      maxPoolSize: 400
+
+Actors:
+- Name: InitialLoad
```

Comment:
Maybe a more descriptive name, such as Load with no secondary indexes. It can be pithier than that.

Reply:
LoadWithOneIndex